.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 
-Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/).
+If the wallet does meet our list policy, please complete the following information which we need to accurately list the wallet.
 
-Only continue with the issue if your wallet meets the criteria listed there.
+### Project info
 
-If it does complete the following information which we need to accurately list the wallet.
+**Wallet name**
 
-**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team or some other method.**
+  <!-- Please provide the official name of the wallet -->
+
+**Wallet description**
+
+<!-- Please provide a short 1-2 sentence description of the wallet without marketing claims. Avoid claims like “the best Ethereum wallet”. We will ask for revisions if this comes across as marketing-focused. -->
+
+**Wallet logo**
+
+<!-- Please provide a hi-res SVG or transparent PNG -->
+
+**Background color for brand logo**
+
+<!-- Please provide a hex code for the brand color. This will be added to the background of the wallet card. -->
+
+**URL to the project**
+
+<!-- Please provide a URL (e.g. to the website of the wallet) -->
+
+**URL to the documentation**
+
+<!-- Please provide a URL to the documentation -->
+
+**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team, or some other method.**
 
 <!-- Please explain any security measures you have taken to ensure your wallet is secure -->
 
 **When did your wallet go live to users?**
 
-<!-- Please provide an exact or approximate date when your wallet was usable by the public -->
+<!-- Please provide a date when your wallet was usable by the public. Please provide some user metrics for how many users are using this wallet. -->
 
 **Does your wallet have an active development team?**
 
-<!-- Are developers actively working on the wallet? -->
-
-**Is your wallet open-source?**
-
-<!-- If yes, please provide a direct link to the repository, if you can -->
+<!-- Are developers actively working on the wallet? Provide proof that the wallet is actively being worked on (ex. GitHub repo link). -->
 
 **Is your wallet globally accessible?**
 
 <!-- Please list any KYC requirements or geographic limitations of your wallet, if any exist. -->
 
-**Is your wallet custodial, non-custodial, or a hardware wallet?**
+**Is there internationalization support for your wallet?**
+
+<!-- Please list languages that the wallet actively supports -->
+
+**What is the repository for your codebase?**
+
+<!-- If you’re project is open source, please provide a link to the codebase. -->
+
+**What social links are there for the project?**
+
+<!-- Please provide social links for the wallet (Discord, Twitter, etc.) -->
+
+**What is your wallet's onboarding experience?**
```

Comment:
I feel like maybe we could drop this question. What insights are we looking to gain from this info?

Reply:
I'm down to have questions that we don't actively use but that allow us to gather insights. However, I would struggle with how to answer this question in its current wording.

Reply:
I think questions that are geared towards us just "gathering more insight" pull us further away from objectively listing products that meet certain criteria. It implies that if our team "doesn't like" your onboarding process that you may not be listed.

If there are objective aspects of the onboarding process that we think are important, I think we should identify and ask about those instead.

Reply:
Will remove this question. Further down we ask for documentation and I think we will get that info there. If anything this is a duplicate that can be removed.